example/lib/sources/complete_form.dart

```diff
@@ -229,7 +229,7 @@ class CompleteFormState extends State<CompleteForm> {
                     ]),
                   ),
                   FormBuilderTextField(
-                    autovalidate: true,
+                    autovalidateMode: AutovalidateMode.always,
```

Comment:
Please consider using capital letter on each substantive 'autoValidateMode'

Reply:
I'm following same naming conventions from the AutovalidateMode enum (done by Flutter team).